### PR TITLE
Feature/lables exclude in value

### PR DESCRIPTION
**Pull request by vl4deee11**
### Type of change
- Enhancement / new feature

### Description

For fluxcd it is useful to exclude labels in values.yaml.

It can be useful with problem https://github.com/strimzi/strimzi-kafka-operator/issues/2558

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

**Comment by vl4deee11** [comment]
I reopened the PR because I broke the last branch a little

**Review by scholzj — APPROVED** [review]

**Comment by scholzj** [comment]
Thanks for the PR

File: helm-charts/helm3/strimzi-kafka-operator/README.md [hunk 1]
```diff
@@ -169,6 +169,7 @@ the documentation for more details.
 | `readinessProbe.periodSeconds`       | Readiness probe period in seconds         | 30                                                   |
 | `imageRegistryOverride`              | Override all image registry config        | `nil`                                                |
 | `imageRepositoryOverride`            | Override all image repository config      | `nil`                                                |
+| `labelsExclusionPattern`             | Override the exclude pattern for exclude some lables             | `""`
```
Comment (ppatierno) [inline helm-charts/helm3/strimzi-kafka-operator/README.md:172]: Typo "labels"

File: helm-charts/helm3/strimzi-kafka-operator/values.yaml [hunk 2]
```diff
@@ -144,3 +144,5 @@ imageRepositoryOverride: ""
 # Override the docker image tag used by all Strimzi images
 imageTagOverride: ""
 createGlobalResources: true
+# Override the exclude pattern for exclude some lables
```
Comment (ppatierno) [inline helm-charts/helm3/strimzi-kafka-operator/values.yaml:147]: Typo "labels"

**Review by ppatierno — APPROVED** [review]
LGTM. Just minor points with typos.

File: helm-charts/helm3/strimzi-kafka-operator/README.md (hunk 1 above)
Comment (vl4deee11) [reply helm-charts/helm3/strimzi-kafka-operator/README.md]: fix

File: helm-charts/helm3/strimzi-kafka-operator/values.yaml (hunk 2 above)
Comment (vl4deee11) [reply helm-charts/helm3/strimzi-kafka-operator/values.yaml]: fix

**Pull request merged**